exporter/googlemanagedprometheusexporter/README.md

```diff
@@ -0,0 +1,77 @@
+# Google Managed Service for Prometheus Exporter
+
+| Status                   |                       |
+| ------------------------ |-----------------------|
+| Stability                | [alpha]               |
```

Comment:
The link is missing. Also, should this be 'in development' instead?

Reply:
changed to in development

Reply:
added link as well